test/core/derivatives/derivatives_test.py

```diff
@@ -210,15 +216,6 @@ def test_weight_jac_mat_prod_should_fail(problem):
         test_weight_jac_mat_prod(problem)
 
 
-@pytest.mark.parametrize(
-    "sum_batch", [True, False], ids=["sum_batch=True", "sum_batch=False"]
-)
-@pytest.mark.parametrize("problem", CONVOLUTION_FAIL_PROBLEMS, ids=CONVOLUTION_FAIL_IDS)
-def test_weight_jac_t_mat_prod_should_fail(problem, sum_batch):
-    with pytest.raises(NotImplementedError):
-        test_weight_jac_t_mat_prod(problem, sum_batch)
-
-
```

Comment:
I think there were also `ConvTranspose` problems in `CONVOLUTION_FAIL_PROBLEMS`. The coverage report should provide more insights